src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -384,136 +370,56 @@ def handlePileup(self):
-        Dataset, divided into blocks, may not have all blocks present on a
-        particular PNN. However, all files belonging into a block will be
-        present when reported by DBS.
-
-        The structure of the pileupDict: PileupFetcher._queryDbsAndGetPileupConfig
-
-        2011-02-03:
-        According to the current implementation of helper testing module
-        WMCore_t/WMRuntime_t/Scripts_t/WMTaskSpace/cmsRun1/PSet.py
-        each type of modules instances can have either "secsource"
-        or "input" attribute, so need to probe both, one shall succeed.
-        """
-        self.logger.info("Requested pileup type %s with %d mixing modules", requestedPileupType, len(modules))
-
-        for m in modules:
-            self.logger.info("Loaded module type: %s", m.type_())
-            for pileupType in self.step.data.pileup.listSections_():
-                # there should be either "input" or "secsource" attributes
-                # and both "MixingModule", "DataMixingModule" can have both
-                inputTypeAttrib = getattr(m, "input", None) or getattr(m, "secsource", None)
-                self.logger.info("pileupType: %s with input attributes: %s", pileupType, bool(inputTypeAttrib))
-                if not inputTypeAttrib:
-                    continue
-                inputTypeAttrib.fileNames = cms.untracked.vstring()
-                if pileupType == requestedPileupType:
-                    eventsAvailable = 0
-                    useAAA = True if getattr(self.jobBag, 'trustPUSitelists', False) else False
-                    self.logger.info("Pileup set to read data remotely: %s", useAAA)
-                    for blockName in sorted(pileupDict[pileupType].keys()):
-                        blockDict = pileupDict[pileupType][blockName]
-                        if PhEDExNodeName in blockDict["PhEDExNodeNames"] or useAAA:
-                            eventsAvailable += int(blockDict.get('NumberOfEvents', 0))
-                            for fileLFN in blockDict["FileList"]:
-                                # vstring does not support unicode
-                                inputTypeAttrib.fileNames.append(str(fileLFN))
-                    if requestedPileupType == 'data':
-                        if getattr(self.jobBag, 'skipPileupEvents', None) is not None:
-                            # For deterministic pileup, we want to shuffle the list the
-                            # same for every job in the task and skip events
-                            random.seed(self.job['task'])
-                            self.logger.info("Skipping %d pileup events for deterministic data mixing",
-                                             self.jobBag.skipPileupEvents)
-                            inputTypeAttrib.skipEvents = cms.untracked.uint32(
-                                int(self.jobBag.skipPileupEvents) % eventsAvailable)
-                            inputTypeAttrib.sequential = cms.untracked.bool(True)
```

Comment:
I might have missed it in the code, is this `sequential` attribute covered in your PR?